site/simple_backup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+echo 'Creating fieldpapers backup'
+today=`date +"%Y-%m-%d"`
+current_dir=`pwd`
+backup_dir="fieldpapers-backup-${today}"
+output_archive="${current_dir}/${backup_dir}.tar.gz"
+
+mkdir -p $backup_dir
+cd $backup_dir
+mkdir -p files
+rsync -r /usr/local/fieldpapers/site/www/files/ ./files/
+mysqldump -u fieldpapers -pw4lks fieldpapers > fieldpapers_db.sql
```

Comment:
It'd be good if you could pull the credentials out of here.  Do you know if MySQL has the equivalent of the Postgres `PGUSER`, etc. environment variables?